services/order.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order
+
+
+@transaction.atomic
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]